src/DssSpell.t.base.sol

```diff
@@ -17,7 +17,7 @@
 pragma solidity 0.8.16;
 
 import "dss-interfaces/Interfaces.sol";
-import {DssTest, GodMode} from "dss-test/DssTest.sol";
+import {DssTest, GodMode, ScriptTools} from "dss-test/DssTest.sol";
```

Comment:
neat - ScriptTools is used only in DssSpell.t.sol so it can be just imported there.